config/initializers/devise.rb

```diff
@@ -44,14 +44,16 @@
 
   # Configure parameters from the request object used for authentication. Each entry
   # given should be a request method and it will automatically be passed to the
-  # find_for_authentication method and considered in your model lookup. For instance,
-  # if you set :request_keys to [:subdomain], :subdomain will be used on authentication.
+  # find_for_authentication method and considered in your model lookup.
```

Comment:
Please do not change comments form external gems if you don't need to. This change no make sense.